lib/Handlers/code/index.ts

```diff
@@ -5,8 +5,10 @@ import safeJSONStringify from 'safe-json-stringify';
 
 import { HandlerFactory } from '@/lib/Handler';
 
+import { vmExecute } from './utils';
+
 export type CodeOptions = {
-  endpoint: string;
+  endpoint: string | null;
 };
 
 const CodeHandler: HandlerFactory<Node, CodeOptions> = ({ endpoint }) => ({
```

Comment:
I think we can set it up so CodeOptions is optional and { endpoint } = { endpoint: null }